auth-web/src/util/config-helper.ts

```diff
@@ -1,18 +1,10 @@
 import { Account, PaymentTypes, SessionStorageKeys } from '@/util/constants'
 
-import Axios from 'axios'
 import CommonUtils from './common-util'
 import { NameRequest } from '@/models/business'
 
-/**
- * the configs are used since process.env doesnt play well when we hae only one build config and multiple deployments..so going for this
- */
-const url = `${process.env.VUE_APP_PATH}config/configuration.json`
-
 export default class ConfigHelper {
   static async fetchConfig () {
-    const response = await Axios.get(url)
-    sessionStorage.setItem(SessionStorageKeys.ApiConfigKey, JSON.stringify(response.data))
```

Comment:
I removed the ApiConfigKey. Can anyone see any issue?

Reply:
only code I see that uses that:

```
  static getValue (key: String) {
    // @ts-ignore
    return JSON.parse(sessionStorage.getItem(SessionStorageKeys.ApiConfigKey))[key]
  }
```